### PR TITLE
Assume positive intent

### PR DESCRIPTION
**Pull request by jansepar**
Something I've been meaning to get in here for a while - we should assume Mobifyers all have positive intentions towards each other!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +214,7 @@ voices are heard.
 > * Be respectful.
 > * Be mindful of the words you choose.
 > * When we disagree, try to understand why.
+> * Assume positive intent
```
Comment (jeremywiebe) [inline README.md]: Sorry to nitpick, but the other points all end in a period. (Yeah, sorry, I'm that guy)

**Comment by ericmuyser** [comment]
Glanced over it, looks good to me!

File: README.md [hunk 2]
```diff
@@ -201,7 +201,7 @@ repeating the mistakes of the past.
 - Go to meetups and conferences and share what you've learned with the rest of the team.
 - Write blog posts or speak at meetups and conferences.
 
-#### Diversity and Collaboration
+#### Promote diversity and collaboration
```
Comment (johnboxall) [inline README.md]: @jansepar prime emoji opportunity. Just saying.

File: README.md (hunk 1 above)
Comment (johnboxall) [inline README.md]: @jansepar I wonder if this is actually the first point on this list.

**Comment by johnboxall** [comment]
👍

**Comment by jansepar** [comment]
Reordered and emoji added!

**Pull request merged**